### PR TITLE
Add io.github.kovzol.bibref

### PR DESCRIPTION
**Pull request by kovzol**
### Please confirm your submission meets all the criteria

Please describe your application briefly below.

**The bibref program is a tool that helps discovering internal references in the Bible. It aims at finding quotations of the Septuagint in the Greek New Testament in a mechanical way.**

- [X] I have read the [App Requirements][reqs] and [App Maintenance][maint] pages.
- [X] My pull request follows the instructions at [App Submission][submission].
- [X] I have [built][build] and tested the submission locally.
- [X] I am using only the minimal set of permissions. *(If not, please explain each non-standard permission.)*
- [X] All assets referenced in the manifest are redistributable by any party.  If not, the unredistributable parts are using an extra-data source type.
- [X] I am an author/developer/upstream contributor of the project. If not, I contacted upstream developers about submitting their software to Flathub. **Link:**
- [X] The domain used for the application ID is controlled by the application developers either directly or through the code hosting (e.g. GitHub, GitLab, SourceForge, etc.). The [application id guidelines][app-id] are followed.
- [X] Any additional patches or files have been submitted to the upstream projects concerned. *(If not, explain why.)*

[reqs]: https://docs.flathub.org/docs/for-app-authors/requirements
[maint]: https://docs.flathub.org/docs/for-app-authors/maintanance
[submission]: https://docs.flathub.org/docs/for-app-authors/submission
[build]: https://docs.flathub.org/docs/for-app-authors/submission/#before-submission
[app-id]: https://docs.flathub.org/docs/for-app-authors/requirements#application-id

### CODE REVIEW

**Comment by hfiguiere** [comment]
Missing

> Please describe your application briefly below.

File: io.github.kovzol.bibref.yaml [hunk 1]
```diff
@@ -0,0 +1,44 @@
+id: io.github.kovzol.bibref
+runtime: org.kde.Platform
+runtime-version: '6.5'
```
Comment (hfiguiere) [inline io.github.kovzol.bibref.yaml:3]: you should use 6.6

File: io.github.kovzol.bibref.yaml [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+    buildsystem: cmake-ninja
+    subdir: qt
+    config-opts:
+      - -DCMAKE_BUILD_TYPE=RelWithDebInfo
+      - -DBUILD_SHARED_LIBS=ON
+      - -UPROJECT_SOURCE_DIR
+      - -DPROJECT_SOURCE_DIR=/app/share/bibref-qt
+    build-options:
+      env:
+        XDG_DATA_HOME: /app/home
```
Comment (hfiguiere) [inline io.github.kovzol.bibref.yaml:36]: I doubt this is correct. It's a readonly location.

File: io.github.kovzol.bibref.yaml [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+    sources:
+      - type: archive
+        url: https://github.com/kovzol/bibref/archive/85f8636bd3ff427c6678022017397bfb42785742.zip
+        sha256: f98542b0f91323627e77bb7839252056c3055c364742d12578a505335367e27d
+  - name: finalize
```
Comment (hfiguiere) [inline io.github.kovzol.bibref.yaml:41]: no reason for this to be a separate module.

File: io.github.kovzol.bibref.yaml [hunk 4]
```diff
@@ -0,0 +1,44 @@
...
+  - name: sword
+    buildsystem: cmake-ninja
+    builddir: true
+    sources:
+      - type: archive
+        url: https://www.crosswire.org/ftpmirror/pub/sword/source/v1.9/sword-1.9.0.tar.gz
+        sha256: 42409cf3de2faf1108523e2c5ac0745d21f9ed2a5c78ed878ee9dcc303426b8a
+  - name: boost
+    buildsystem: simple
+    build-commands: ['./bootstrap.sh --prefix=/app --with-libraries=system,filesystem', './b2 -j $FLATPAK_BUILDER_N_JOBS install']
```
Comment (hfiguiere) [inline io.github.kovzol.bibref.yaml:21]: please use proper yaml syntax. it's unreadable otherwise.

File: io.github.kovzol.bibref.yaml (hunk 1 above)
Comment (kovzol) [reply io.github.kovzol.bibref.yaml:3]: Thanks, I'll update the version.

File: io.github.kovzol.bibref.yaml (hunk 2 above)
Comment (kovzol) [reply io.github.kovzol.bibref.yaml:36]: The program puts some data in this folder. I will change this to be more readable, but technically the new version will do the same.

File: io.github.kovzol.bibref.yaml (hunk 3 above)
Comment (kovzol) [reply io.github.kovzol.bibref.yaml:41]: I only need an extra command at the end, but if I put it in the previous module, it overwrites its build command.

File: io.github.kovzol.bibref.yaml (hunk 4 above)
Comment (kovzol) [reply io.github.kovzol.bibref.yaml:21]: I will fix this in a new commit.

File: io.github.kovzol.bibref.yaml (hunk 2 above)
Comment (hfiguiere) [reply io.github.kovzol.bibref.yaml:36]: it can't put anything. it's read only.,

File: io.github.kovzol.bibref.yaml (hunk 3 above)
Comment (hfiguiere) [reply io.github.kovzol.bibref.yaml:41]: `post-install` is here for that.

File: io.github.kovzol.bibref.yaml (hunk 2 above)
Comment (kovzol) [reply io.github.kovzol.bibref.yaml:36]: Yes, for the application it's read only (that's okay), but not for the build script. The build script has full access to /app/home.

File: io.github.kovzol.bibref.yaml (hunk 3 above)
Comment (kovzol) [reply io.github.kovzol.bibref.yaml:41]: I see, thanks!

**Comment by bbhtt** [comment]
bot, build io.github.kovzol.bibref

**Pull request closed without merging**